enumerables.rb

```diff
@@ -0,0 +1,160 @@
+module Enumerable
+
+  # my_each
+  # rubocop:disable Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
+  def my_each
+    return to_enum(:my_each) unless block_given?
+      i = 0
+      while i < self.size do
+        yield(self.to_a[i])
+        i += 1
+      end
+  end
```

Comment:
[Optional]

Return `self` too here as explained in the previous comment about `my_each_with_index`